hardhat.config.ts

```diff
@@ -21,6 +21,16 @@ const config: HardhatUserConfig = {
   },
   solidity: {
     compilers: [
+      {
+        version: '0.8.2',
+        settings: {
+          optimizer: {
+            enabled: true,
+            // @review
+            runs: 1,
```

Comment:
This optimizes for bytecode-size at the expense of runtime gas-cost for function calls. It seems like maybe a reasonable tradeoff given our focus on moving Assets to L2.